alacritty.yml

```diff
@@ -524,67 +530,73 @@
 #   - ReceiveChar
 #   - None
 #
-#   (`mode: Vi` only):
+# - Vi mode exclusive actions:
+#
 #   - Open
-#       Open URLs at the cursor location with the launcher configured in `url.launcher`.
+#       Open URLs at the cursor location with the launcher configured in
+#       `url.launcher`.
+#   - ToggleNormalSelection
+#   - ToggleLineSelection
+#   - ToggleBlockSelection
+#   - ToggleSemanticSelection
+#       Toggle semantic selection based on `selection.semantic_escape_chars`.
+#
+# - Vi mode exclusive cursor motion actions:
```

Comment:
I've made the choice of separating different actions a bit more, to make it possible to shorten the descriptions of the actions dramatically. That should hopefully make things a little quicker to read.

I personally find that the different sections of the bindings are a bit difficult to see though, so maybe there's a better way to do that.

Maybe we should use markdown style `# Headings` instead of the `- Sections:` we have right now?

Reply:
I think it could confuse a bit, since `#` is a start of comment in yaml? Though, I'm not completely against `#` for certain things... Up to you here.

Reply:
Yeah I'm not sure if I want to change anything about that at this point. Just an idea for the future.